yt/fields/angular_momentum.py

```diff
@@ -26,78 +26,84 @@
 from .vector_operations import \
     create_magnitude_field
 
-from yt.utilities.lib.geometry_utils import \
+from yt.utilities.lib.misc_utilities import \
```

Comment:
This isn't necessarily the best line to leave this comment, but can `obtain_rvec` and `obtain_rv_vec` be given more descriptive names?  I had in my head the whole time that `obtain_rvec` signified "radial vector" and subsequently that `obtain_rv_rec` meant "radial velocity", but it's relative velocity.  I could see how others might also be confused by this.

Reply:
How about `obtain_position_vector` and `obtain_relative_velocity_vector`?

Reply:
Yeah, I'm totally fine with those.